specs/data_structures.md

```diff
@@ -800,6 +800,18 @@ enum AccountStatus : uint8_t {
 | `nonce`   | [Nonce](#type-aliases)  | Account nonce. Every outgoing transaction from this account increments the nonce. |
 | `status`  | `AccountStatus`         | Validator or delegation status of this account.                                   |
 
+The `status` of an account indicates weather it is a validator (`AccountStatus.Validator*`), delegating to a validator (`AccountStatus.Delegation*`), or neither (`AccountStatus.None`). Being a validator and delegating are mutually exclusive, and only a single validator can be delegated to.
+
```

Comment:
Are the implications of this mentioned somewhere too? E.g., that you need to create another account per delegation (and that this doesn't cause the UX to get worse as the wallet/UI can simply use a different derivation path per delegation etc)?

Reply:
No. Not sure if that belongs directly in the technical specs, but if it's something that comes up we can probably write a rationale document around our staking design later.